commands/audit/audit_test.go

```diff
@@ -149,3 +144,54 @@ func TestAuditWithConfigProfile(t *testing.T) {
 		})
 	}
 }
+
+// This test tests audit flow when providing --output-dir flag
+func TestAuditWithScansOutputDir(t *testing.T) {
+	// TODO this test needs to be improved to verify the files content after we use the Sarif convertor and write Sarif content into the files
```

Comment:
remove log, when adding the feature we will change the test since it will not pass anymore